README.md

```diff
@@ -44,15 +44,33 @@ To deactivate the virtualenv (after you finished working):
 Install dependencies in virtual environment:
 
     $ pip3 install -r requirements.txt
-    
+
+## Environment Variables
+
+TBD
+
 ## Run Application
 
 Start the server by running:
-  
-    $ python web/app.py
 
-## Unit Test
+    $ export FLASK_ENV=development
```

Comment:
I upgraded the version of flask to 1.0, and this is the new recommended way to run the server.  (This has the advantage of reloading the server whenever you edit the source code.)